### PR TITLE
demo: update Tag demo

### PR DESCRIPTION
**Pull request by li-jia-nan**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [x] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | demo: update Tag demo |
| 🇨🇳 Chinese | 删除了多余的Space，并且删除了最后一个元素的右边距 |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

---

### <samp>🤖 Generated by Copilot at d88d6c5</samp>

This pull request improves the code and appearance of the Tag component demo. It sorts the imports, removes an unnecessary Space component, and fixes a margin bug in the `.code-box` class.

### <samp>🤖 Generated by Copilot at d88d6c5</samp>

* Fix a visual bug in the Tag component demo by adjusting the margin of the last Tag element in the Space component ([link](https://github.com/ant-design/ant-design/pull/44771/files?diff=unified&w=0#diff-6d4081217418af981e22c0fd474410c584b0d3c238609e557164b0ebd07c9ab4R492-R505))
* Simplify the Tag component demo code by removing an unnecessary nested Space component ([link](https://github.com/ant-design/ant-design/pull/44771/files?diff=unified&w=0#diff-71f7311986ed1c781a47dc64541789fc8eac0682d3369908ec75c984f36055e4L75-R134))
* Improve the code readability and style consistency of the `components/tag/demo/control.tsx` file by reordering the import statements according to the ESLint rule `sort-imports` ([link](https://github.com/ant-design/ant-design/pull/44771/files?diff=unified&w=0#diff-71f7311986ed1c781a47dc64541789fc8eac0682d3369908ec75c984f36055e4L4-R4))

### CODE REVIEW

File: .dumi/theme/common/styles/Markdown.tsx [hunk 1]
```diff
@@ -489,6 +489,20 @@ const GlobalStyle: React.FC = () => {
...
+              ${antCls}-space-item {
+                &:last-of-type {
+                  ${antCls}-tag {
+                    margin-inline-end: 0;
+                  }
+                }
+              }
+            }
+          }
+        }
```
Comment (afc163) [inline .dumi/theme/common/styles/Markdown.tsx:505]: 为啥要加这个，感觉有点难受

Comment (li-jia-nan) [reply .dumi/theme/common/styles/Markdown.tsx:505]: 最后一个元素有个多余的右边距：

image

File: components/tag/demo/control.tsx [hunk 2]
```diff
@@ -72,68 +72,66 @@ const App: React.FC = () => {
...
+          size="small"
+          style={tagInputStyle}
+          value={inputValue}
+          onChange={handleInputChange}
+          onBlur={handleInputConfirm}
+          onPressEnter={handleInputConfirm}
+        />
+      ) : (
+        <Tag style={tagPlusStyle} onClick={showInput}>
+          <PlusOutlined /> New Tag
```
Comment (afc163) [inline components/tag/demo/control.tsx:132]: Tag 有 icon 属性。

File: .dumi/theme/common/styles/Markdown.tsx (hunk 1 above)
Comment (afc163) [reply .dumi/theme/common/styles/Markdown.tsx:505]: 这个问题不大，建议不改。

Comment (li-jia-nan) [reply .dumi/theme/common/styles/Markdown.tsx]: 删掉了

File: components/tag/demo/control.tsx (hunk 2 above)
Comment (li-jia-nan) [reply components/tag/demo/control.tsx]: done~

**Review by afc163 — APPROVED** [review]

**Pull request merged**